Automattic/jetpack
# AI Extension: tweak the tone prompt

Fixes #

## Proposed changes:

* AI Extension: tweak the tone prompt

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Get some content by using a paragraph block instance. 
* Change to a `confident` tone
* Translate to another language
* Change to a `confident` tone, again
* Confirm the generated content preserves the language.
